src/app/personside/visittkort/header/NavKontorContainer.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { NavKontorInterface } from '../../../../models/navkontor';
+import { AppState, Reducer } from '../../../../redux/reducer';
+import Innholdslaster from '../../../../components/Innholdslaster';
+import Undertekst from 'nav-frontend-typografi/lib/undertekst';
+import styled from 'styled-components';
+
+interface Props {
+    navKontorReducer: Reducer<NavKontorInterface>;
+}
+
+const CenterVertically = styled.div`
+  display: inline-flex;
+  align-items: center;
+`;
+
+const onError = (
+    <em>Problemer med å hente nav-enhet</em>
```

Comment:
Hva gjør em her? Italic?

Reply:
Kan også skrive NAVmed store bokstaver :)

Reply:
emphasis. tror den blir italic ja, men tanken er vel at man skal omslutte ting som skal fremheves med em, også tar global-styling/nettleseren seg av resten? Ikke helt sikker, men kom gjerne med et bedre forslag hvis du har noen ideer ;-) Prøvde med alertstripe, men den blir altfor stor og dominerende

Reply:
ser at nav er skrevet Nav i skissen, går for det for øyeblikket